### PR TITLE
UGent news

### PR DESCRIPTION
**Pull request by draxaris1010**

### CODE REVIEW

**Comment by draxaris1010** [comment]
Er is wat tijdelijke code om de taal van het OS te gebruiken in [src/stores/news.ts#L14-L19](https://github.com/ZeusWPI/hydra-react-native/pull/22/files#diff-2c484b486775d58f39cea5e284de9dfca822067ab5bf1fb101e9d59938bf1120R14-R19)

File: src/stores/news.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  return useSuspenseQuery<UGentArticle[], Error>({
+    queryKey: ["news"],
+    queryFn: async () => {
+
+      // This will later be managed by the settings in the app
+      var language = 'en';
+      if((Platform.OS == "android" && NativeModules.I18nManager.localeIdentifier.startsWith("nl")) 
+      || (Platform.OS == "ios" && NativeModules.SettingsManager.settings.AppleLocale.startsWith("nl"))) {
+        language = 'nl';
+      }
```
Comment (NuttyShrimp) [inline src/stores/news.ts:19]: Replace this with `https://docs.expo.dev/guides/localization/#getting-the-users-language`. If you rebase the branch on master, all dependencies should be there for you

File: src/routes/news.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+// import { ScrollView } from "react-native-gesture-handler";
+import { useUGentQuery } from "../stores/news";
+import { UGentArticleFeed } from "../components/feed/UGentArticle";
+import { StyleSheet, ScrollView } from "react-native";
```
Comment (NuttyShrimp) [inline src/routes/news.tsx:4]: Import `ScrollView` from `react-native-gesture-handler`

**Review by NuttyShrimp — CHANGES_REQUESTED** [review]

File: src/stores/news.ts (hunk 1 above)
Comment (draxaris1010) [reply src/stores/news.ts]: Done.

File: src/routes/news.tsx (hunk 2 above)
Comment (draxaris1010) [reply src/routes/news.tsx]: Done.

**Review by NuttyShrimp — APPROVED** [review]

**Pull request merged**